modules/server/src/main/ts/bedrock/auto/DriverLoader.ts

```diff
@@ -102,78 +62,29 @@ export const loadDriver = (browserName: string, settings: DriverSettings): Drive
   }
 
   const execNames = browserExecutables[browserName] || [];
-  const execPath = findExecutable(execNames);
+  const execPath = ExecUtils.findExecutable(execNames);
   if (execPath !== null) {
-    return execLoader(execPath);
+    return ExecUtils.execLoader(execPath);
   } else {
-    throw new Error('Unable to find a suitable driver for ' + browserName);
+    throw new Error('Local1: Unable to find a suitable driver for ' + browserName);
```

Comment:
`Local1` ?  What is that?

Reply:
Was used for local debugging. Has been removed